Posts/postsrouters.js

```diff
@@ -0,0 +1,159 @@
+const express = require('express');
+const usersDb = require('../data/helpers/userDb');
+const postsDb = require('../data/helpers/postDb');
+const errorHandler = require('../ErrorHandler/errorhandler');
+const router = express.Router();
+
+// TODO: Stretch: Abstract out error handling
+const usersDbAccessError = {"error": "There was an error accessing the users database table."};
+const postsDbAccessError = {"error": "There was an error accessing the posts database table."};
+
+// ~~ GET ~~ //
+// postDB: get() -> [{obj1},...,{objN}]
+router.get('/', (req, res) => {
+    postsDb.get()
+        .then((postsList) => {
+            res.status(200).json(postsList);
+        })
+        .catch((err) => {
+            console.error('postDb.js/get() Access Error:\n', err);
+            res.status(500).json(postsDbAccessError);
+        });
+});
+
+// postDB: get(id) -> {obj}
+router.get('/:id', (req, res) => {
+    postsDb.get(req.params.id)
+        .then((post) => {
+            if(post !== undefined) {
+                res.status(200).json(post);
+            } else {
+                res.status(404).json({"error": `A post with ID ${req.params.id} doesn't exist.`});
+            }
+        })
+        .catch((err) => {
+            console.error('postDb.js/get(id) Access Error:\n', err);
+            res.status(500).json(postsDbAccessError);
+        });
+});
+
+// ~~ POST ~~ /
+// postDB: insert({obj}) -> {id: ##}
+router.post('/', (req, res) => {
+    if(req.body.userId && req.body.text) {
```

Comment:
You should assign `req.body.userId` and `req.body.text` to variables since you're using  them more than once. It'll pay off in the long run.